### PR TITLE
Remove unnecessary panics from polybft

### PR DESCRIPTION
**Pull request by Stefan-Ethernal**
# Description

This PR removes panics from the polybft package and adds error handling instead.

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the relevant labels
- [ ] I have updated the official documentation
- [ ] I have added sufficient documentation in code

## Testing

- [ ] I have tested this code with the official test suite
- [ ] I have tested this code manually

### CODE REVIEW

**Review by 0xSasaPrsic — APPROVED** [review]
Looks good.

File: consensus/polybft/blockchain_wrapper.go [hunk 1]
```diff
@@ -196,5 +201,5 @@ func (s *stateProvider) Call(addr ethgo.Address, input []byte, opts *contract.Ca
 // Txn is part of the contract.Provider interface to make Ethereum transactions. We disable this function
 // since the system state does not make any transaction
 func (s *stateProvider) Txn(ethgo.Address, ethgo.Key, []byte) (contract.Txn, error) {
```
Comment (igorcrevar) [inline consensus/polybft/blockchain_wrapper.go:203]: I would leave panic here, this is just to notify developer that this object should not be used for something for which is not meant

**Review by igorcrevar — APPROVED** [review]
I would not remove panic from Txn method but otherwise good -> approved

**Review by Nemanja0x — APPROVED** [review]

File: consensus/polybft/blockchain_wrapper.go (hunk 1 above)
Comment (ferranbt) [reply consensus/polybft/blockchain_wrapper.go:203]: I agree with @igorcrevar. This can never happen.

**Review by ferranbt — APPROVED** [review]

File: consensus/polybft/blockchain_wrapper.go (hunk 1 above)
Comment (Stefan-Ethernal) [reply consensus/polybft/blockchain_wrapper.go:203]: Fixed https://github.com/0xPolygon/polygon-edge/pull/1077/commits/9f8f1c2022511ec0bc37851484a86adc3bd311ca

**Pull request merged**